### ISSUE TITLE
`Filament Fabricator: Layout "default" not found` when using Octane

### ISSUE THREAD

**Issue by plunkettscott** [Z3d0X/filament-fabricator#120]
When using Octane (tried both Swoole and RoadRunner for confirmation), pages occasionally load correctly on the first request, but it seems that subsequent requests fail with an error about the layout missing. The same site works correctly without Octane.

**Comment by Z3d0X** [comment]
Potentially related to #111

**Comment by ksimenic** [comment]
I have the same problem and managed to get to the root of it.

The issue is in `Z3d0X\FilamentFabricator\FilamentFabricatorServiceProvider`.
```
    public function packageRegistered(): void
    {
        parent::packageRegistered();

        $this->app->scoped('filament-fabricator', function () {
            return new FilamentFabricatorManager();
        });
    }
```
Based on the https://laravel.com/docs/10.x/container#binding-scoped 

> The scoped method binds a class or interface into the container that should only be resolved one time within a given Laravel request / job lifecycle. While this method is similar to the singleton method, instances registered using the scoped method will be flushed whenever the Laravel application starts a new "lifecycle", such as when a [Laravel Octane](https://laravel.com/docs/10.x/octane) worker processes a new request or when a Laravel [queue worker](https://laravel.com/docs/10.x/queues) processes a new job:

Because `filament-fabricator` is registered as `scoped` it is flushed on every call. When it instantiates `new FilamentFabricatorManager` this is what happens in the constructor

```
    public function __construct()
    {
        /** @var Collection<string,string> */
        $pageBlocks = collect([]);

        /** @var Collection<string,string> */
        $layouts = collect([]);

        $this->pageBlocks = $pageBlocks;
        $this->layouts = $layouts;
    }
```

meaning all previously registered page blocks and layouts will be overwritten by empty collections (removed).

Now this would be fine if `bootingPackage()` method from `Z3d0X\FilamentFabricator\FilamentFabricatorServiceProvider` is called, but this happens only once.

This will eventually result in having page blocks and layouts visible on first and N-1 subsequent requests (N = number of workers that you are running octane with), but on the N+1 request they will simply disappear and be gone until you restart octane server.

Solution to this problem is simply changing `scoped` with `singleton` but I am not 100% sure of the full implications this might have on the package.

```
    public function packageRegistered(): void
    {
        parent::packageRegistered();

        $this->app->singleton('filament-fabricator', function () {
            return new FilamentFabricatorManager();
        });
    }
```

What was the reason of going with `scoped` instead of `singleton` in the first place @Z3d0X?

**Comment by Z3d0X** [comment]
I don't think there was any specific reason for going with `scoped` instead of `singleton`. 

If `singleton` works let's change to that. Thanks for investigating this issue.

**Comment by plunkettscott** [comment]
@Z3d0X  any idea when this will be tagged?

**Comment by Z3d0X** [comment]
`v2.0.6` Released with #130
